dashboard-app/src/util/dateRange.ts

```diff
@@ -0,0 +1,49 @@
+import moment from 'moment';
+
+const staticMonths = [
```

Comment:
I think eventually we should aim to make this more generic and move it to `twine-util`, but not right now.

Reply:
agreed. I like the habit of keeping utils local to a repo until they are required by another app.